control_panel_api/tools.py

```diff
@@ -119,6 +120,32 @@ def deploy_params(self, user):
         ]
 
 
+class Airflow(BaseTool):
+    name = 'airflow-sqlite'
+
+    def release_name(self, username):
+        return f'{username}-{self.name}'
```

Comment:
This override does nothing - you can remove it

Reply:
removed